src/firefly/js/visualize/ui/MouseReadout.jsx

```diff
@@ -30,75 +33,249 @@ var rS= {
-					 <div style={ textStyle} >{updateField('imagePixel' )} </div>
-					 {showReadout(plot, mouseState, CoordinateSys.PIXEL)}
-				 </div>
-		    </div>
+
+			  <div>
+				  <div style={leftColumn} > {showReadout(plot, mouseState,visRoot.flux ) } </div>
+				  <div style={ middleColumn}  onClick={ () => showDialog('readout2' ,visRoot.mouseReadout2)}>
+					 <div style={ textStyle} >{getLabel( visRoot.mouseReadout2)} </div>
+					 {showReadout(plot, mouseState, visRoot.mouseReadout2)}
+				  </div>
+				  <div style={rightColumn} title='Click on an image to lock the display at that point.'   >
+					  <input type='checkbox' name='aLock' value='lock'
+							 onChange = { (request) => setClickLock(plot,mouseState , request) } />
+					  Lock by click
+				  </div>
+		      </div>
 
 		  </div>
 
 	);
 }
 
-function showReadout(plot, mouseState, coordinate){
-	if (!plot) return'';
-	if (isBlankImage(plot)) return'';
+MouseReadout.propTypes= {
+	visRoot:React.PropTypes.object.isRequired,
+	plotView: React.PropTypes.object,
+	mouseState:React.PropTypes.object.isRequired,
+};
+
+function renderEmpty(plotView,plot, mouseState, visRoot){
+	var leftColumn = {width: 120, display: 'inline-block'};
+
+	var title = plotView.plots[0].title;
+	var middleColumn = {width: 250, display: 'inline-block'};
+	var  textStyle = {textDecoration: 'underline', color: 'DarkGray', fontStyle:'italic' ,  display: 'inline-block'};
+	var rightColumn = {paddingLeft: '35px',  display: 'inline-block'};
+	return (
+		<div style={ rS}>
+			<div>
+				<div style={leftColumn} onClick={ () => showDialog('pixelSize', visRoot.pixelSize)}>
+					<div style={ textStyle} > {getLabel(visRoot.pixelSize) }</div>
+					{EMPTY}
+				</div>
+
+				<div style={middleColumn} onClick={ () => showDialog('readout1' ,visRoot.mouseReadout1)}>
+					<div style={ textStyle} > { getLabel( visRoot.mouseReadout1) } </div>
+					{EMPTY}
+				</div>
+
+				<div style={rightColumn}> {title}  </div>
+			</div>
+
+			<div>
+				<div style={leftColumn} >  </div>
+				<div style={ middleColumn}  onClick={ () => showDialog('readout2' ,visRoot.mouseReadout2)}>
+					<div style={ textStyle} >{getLabel( visRoot.mouseReadout2)} </div>
+					{EMPTY}
+				</div>
+				<div style={rightColumn} title='Click on an image to lock the display at that point.'   >
+					<input type='checkbox' name='aLock' value='lock'
+						   onChange = { (request) => setClickLock(plot,mouseState , request) } />
+					Lock by click
+				</div>
+			</div>
+
+		</div>
+
+	);
+}
+function getLabel(radioValue){
+	var gLabel;
+	switch(radioValue){
+		case 'eqj2000hms':
+		case 'eqj2000DCM':
+			gLabel='EQ-J2000:';
+			break;
+		case 'galactic':
+			gLabel='Gal:';
+			break;
+		case 'eqb1950':
+			gLabel='Eq-B1950:';
+			break;
+		case 'fitsIP':
+			gLabel='Image Pixel:';
+			break;
+		case 'pixelSize':
+			gLabel='Pixel Size:';
+			break;
+		case 'sPixelSize':
+			gLabel='Screen Pixel Size:';
+			break;
+	}
+    return gLabel;
+}
```

Comment:
Why don't you make an const object that has this mapping.  I think it would be cleaner than a switch.